docs/how-to-guides/web-portal/web-portal-24-04-or-later/use-security-profiles.md

```diff
@@ -0,0 +1,74 @@
+(how-to-web-portal-use-security-profiles)=
+# How to use security profiles
+
+You can use security profiles to automatically audit and fix your managed instances on a schedule using the [Ubuntu Security Guide (USG)](https://documentation.ubuntu.com/security/docs/compliance/usg/).
+
+```{note}
+This feature is only available in self-hosted **Landscape 25.04** and later.
+```
+
+## Create a security profile
+
+On each instance you want to manage, follow the steps in [Install Ubuntu Security Guide](https://documentation.ubuntu.com/security/docs/compliance/usg/install-usg/), then:
+
+1. Edit the client configuration file `/etc/landscape/client.conf`, add this line:
+
+    ```
+    include_manager_plugins = UsgManager
+    ```
+
+    If your `client.conf` already includes an `include_manager_plugins` line, then add `UsgManager` to it. For example:
+
+    ```
+    include_manager_plugins = ScriptExecution,UsgManager
+    ```
+
+2. Restart Landscape Client:
+
+    ```
+    sudo systemctl restart landscape-client
+    ```
+
+From the web portal:
+
+1. Click **Profiles**
+2. Click **Security Profiles**
+3. Click **Add security profile**
+
+In the security profile creation form, complete the following fields:
+
+- **Profile name**: name of the profile
+- **Access group**: the access group the profile will apply to
+- **Base profile**: the security benchmark the profile will use
+- **Mode**: the profile's mode – which actions it will perform on instances
```

Comment:
```suggestion
- **Access group**: The access group the profile will apply to
- **Base profile**: The security benchmark the profile will use
- **Mode**: The profile's mode – which actions it will perform on instances
```